packages/@aws-cdk/tmp-toolkit-helpers/test/api/diff/diff.test.ts

```diff
@@ -59,20 +59,18 @@ describe('formatStackDiff', () => {
     // WHEN
     const formatter = new DiffFormatter({
       ioHelper: mockIoHelper,
-      oldTemplate: {},
-      newTemplate: {
-        template: {},
-        templateFile: 'template.json',
+      templateInfo: {
+        oldTemplate: {},
+        newTemplate: {
+          template: {},
+          templateFile: 'template.json',
+          stackName: 'test-stack',
+          findMetadataByType: () => [],
+        } as any,
```

Comment:
there are proper helpers for this around

Reply:
why not `mockNewTemplate`?

Reply:
oops